x-pack/platform/plugins/private/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -107,7 +108,7 @@ export const TransformManagement: FC = () => {
   const styles = useStyles();
   const [transformPageState, setTransformPageState] = usePageUrlState<PageUrlState>(
     'transform',
-    getDefaultTransformListState()
+    useUrlState('_a')?.[0] || getDefaultTransformListState()
```

Comment:
can you kindly help me understand how `@kbn/rison` encoded string is compatible with eui's `EuiSearchBar.Query.parse(searchQueryText)` https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/private/transform/public/app/sections/transform_management/components/transform_list/transform_list.tsx#L152